ObliviousHarmony/vscode-php-codesniffer
# Prepare 1.6.0 Release

### All Submissions:


### Changes proposed in this Pull Request:

This commit prepares `main` for the 1.6.0 release. It updates the version as well as all of the dependencies.

### How to test the changes in this Pull Request:

1. Run `vsce package`
2. Confirm package is version 1.6.0
3. Confirm package works
